README.md

```diff
@@ -20,53 +18,15 @@
 
-"To be written ..."
-
-## Background
-* "To be written ..."
-
-
-
-## Meetings
-* GMD working group meets every Wednesday at 11am CST (usually 18:00 CET, but beware different switches to Summer time in EU and US, [check your local time](http://www.thetimezoneconverter.com/?t=11am&tz=Chicago&)) in the CHAOSS Zoom room https://unomaha.zoom.us/j/720431288 (Meeting ID for [dial in](https://unomaha.zoom.us/zoomconference?m=DKGo2mmIuOv9xSjphoGZZmYKxr5HFrS9): 720 431 288). 
-[Meeting Notes from previous meetings are available here.](./meeting_notes.md)
-
-Usually, we try to make decissions mainly during the last meeting of each month,
-so that decisions can be brought when convenient to the monthtly main CHAOSS meeting,
-which is the first meeting of each month. In all the meetings we may have an agenda,
-but random issues can be raised if time allows, after we're done with the agenda.
-Whenever possible, and specially if a decission needs to be made,
-ensure that issues / pull requests about the subject matter were opened some time before the meeting,
-so that anyone had the opportunity of commenting on them, and make up their mind about that subject matter.
-
-**Exceptions** (these days there will be no meeting): none for now
-
-
-Focus areas for this working group include metrics related to ;
-1. Growth Maturity and Decline
-  * [Code Code Development](./focus_areas/code_development.md)
-  * [Community Growth](./focus_areas/community_growth.md)
-  * [Issue Resolution](./focus_areas/issue_resolution.md)
-2. [Risk](./focus_areas/risk/risk.md)
-3. [Value](./focus_areas/value/value.md)
-
-[gmd]: gmd_metrics.md
-
-
-### Join our Discussion Forum
-* We use the [CHAOSS mailing list](https://chaoss.community/participate/#user-content-join-the-mailing-list).
-Please prefix the message subject with \[wg-gmd\] if it is specific to this working group.
-
-
-
 ## Contributing
-* You can start by introducing yourself on the CHAOSS mailing list (see below) explaining your interest. Then, you can have a look at the archives of the mailing lists, at the minutes of past meetings, and at the
-- [issues](https://github.com/chaoss/wg-gmd/issues) and
-- [pull requests](https://github.com/chaoss/wg-gmd/pulls) in
-- [this repository](https://github.com/chaoss/wg-gmd).
 
-Then, of course you can participate in the mailing list, in online meetings, and in issues and pull requests. [Mailing lists can be subscribed to here.](https://lists.linuxfoundation.org/mailman/listinfo/chaoss)
+You can start by introducing yourself on the CHAOSS mailing list (see below) explaining your interest. Then, you can have a look at the archives of the mailing lists, at the minutes of past meetings, and at the
```

Comment:
line 23: "on the CHAOSS mailing list (see below)" --- I suggest turning this into a link, rather then send a reader on a quest to find the section.